setup.py

```diff
@@ -33,6 +33,25 @@
 #
 
 
+def extract_req(req):
+    req = req.strip().split(";")
+    if len(req) > 1:
+        env_marker = req[-1]
+        env_marker = env_marker.replace(" ", "").strip()
+        # for now we just recognize sys_platform
+        if "sys_platform==" in env_marker:
+            target = env_marker.split("!=")[-1].strip()
```

Comment:
This line has `"!=` but the `if` conditional above has `==`.  Bug?

Reply:
yeah that's a typo, good catch thanks!